src/clr-angular/forms/styles/containers.spec.ts

```diff
@@ -277,7 +277,7 @@ describe('Form layouts', () => {
     const computed = height(selector);
     expect(Math.round(computed)).toEqual(Math.round(value));
     if (verticalRhythm) {
-      expect(value % 6).toEqual(0);
+      expect(value % base).toEqual(0);
```

Comment:
After making the change above, many of these tests should pass again without changes.